packages/fuel-indexer-tests/tests/graphql_server.rs

```diff
@@ -256,24 +259,6 @@ async fn test_filtering() {
 
     mock_request("/ping").await;
 
-    // ID selection
```

Comment:
- This part of the test was giving problems
- Something tells me it's related to passing the `ID` in here manually where `ID` was simply 1 - however, now, ID is now calculated differently